news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 43 of 2018"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2018-10-31 14:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase during the last week, from Monday 22nd to Sunday 28th of October 2018.
+
+![Core Weekly banner](/assets/images/2017/04/core_weekly_banner.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week, part of the PrestaShop team's developers attended the PHP Forum in Paris. It is a good event to discover what's new with the technology and learn new skills. However, it has an impact on the productivity: less pull requests have been merged than usual.
+
+The recent upgrade of PrestaShop's APIs infrastucrures improved the overall performance of stores back office. In parallel, the Gamification module has been improved to. A new release is available, for both 1.6 and 1.7 versions of PrestaShop:
+
+* All static files are included in the module, in order to remove a lot of requests to the servers
+* The json data requested by the module is 90% lighter in order to decrease the bandwidth
```

Comment:
As far as i know, the JSON is the same but its content is compressed in the answer.